test/connectors/PEGAConnectorSpec.scala

```diff
@@ -112,14 +112,14 @@ class PEGAConnectorSpec extends SpecBase with FeatureSwitching {
         )
       )
       val result: AppealSubmissionResponse = await(connector.submitAppeal(modelToSend,
-        "HMRC-MTD-VAT~VRN~123456789", isLPP = true, penaltyId = "1234567890", correlationId = "id"))
+        "HMRC-MTD-VAT~VRN~123456789", isLPP = true, penaltyNumber = "1234567890", correlationId = "id"))
       result shouldBe Right(appealResponseModel)
     }
   }
 
   "headerForEIS" should {
     "return a HeaderCarrier with the correct headers" in new Setup {
-      val result: HeaderCarrier = connector.headersForEIS("id", "token", "env")
+      val result: HeaderCarrier = connector.headersForEIS("id", "token", "env", "penaltyNumber")
       result.otherHeaders.toMap.get("Environment").get shouldBe "env"
       result.otherHeaders.toMap.get("CorrelationId").get shouldBe "id"
       result.otherHeaders.toMap.get(AUTHORIZATION).get shouldBe "Bearer token"
```

Comment:
Could you add another test to make sure penaltyNumber is being sent in the headers?